tests/api/saml/fixtures.py

```diff
@@ -711,27 +711,27 @@
         <ds:KeyInfo>
           <ds:X509Data>
             <ds:X509Certificate>
-{19}
+{}
             </ds:X509Certificate>
           </ds:X509Data>
         </ds:KeyInfo>
       </KeyDescriptor>
-      <NameIDFormat>{20}</NameIDFormat>
-      <NameIDFormat>{21}</NameIDFormat>
+      <NameIDFormat>{}</NameIDFormat>
+      <NameIDFormat>{}</NameIDFormat>
       <SingleSignOnService
         Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST-SimpleSign"
         Location="http://idp.hilbertteam.net/idp/profile/SAML2/POST-SimpleSign/SSO"/>
       <SingleSignOnService
-        Binding="{22}"
-        Location="{23}"/>
+        Binding="{}"
+        Location="{}"/>
     </IDPSSODescriptor>
     <Organization>
-      <OrganizationName xml:lang="en">{24}</OrganizationName>
-      <OrganizationName xml:lang="es">{25}</OrganizationName>
-      <OrganizationDisplayName xml:lang="en">{26}</OrganizationDisplayName>
-      <OrganizationDisplayName xml:lang="es">{27}</OrganizationDisplayName>
-      <OrganizationURL xml:lang="en">{28}</OrganizationURL>
-      <OrganizationURL xml:lang="es">{29}</OrganizationURL>
+      <OrganizationName xml:lang="en">{}</OrganizationName>
+      <OrganizationName xml:lang="es">{}</OrganizationName>
+      <OrganizationDisplayName xml:lang="en">{}</OrganizationDisplayName>
+      <OrganizationDisplayName xml:lang="es">{}</OrganizationDisplayName>
+      <OrganizationURL xml:lang="en">{}</OrganizationURL>
+      <OrganizationURL xml:lang="es">{}</OrganizationURL>
```

Comment:
While I agree that the new form of this is very hard to read, I don't think that it is any worse than the previous form. In one way, I find it slightly better: by not using an index into the `format` args, the change makes it clear that none of the arguments are used out of order or in more than one place.

I think the right way to fix this to switch to f-strings for interpolation, so that we can see the symbolic name inline.